src/ServicePulse.Host/app/js/services/factory.rxjs.js

```diff
@@ -0,0 +1,8 @@
+; (function (window, angular, undefined) {
+    'use strict';
+
+    angular.module('sc')
+        .service('rx', function () {
```

Comment:
Good point. In all honesty, we're just paying lipservice to injecting a dependency here. Why not simply use RxJs without declaring it as an angular dependency? We'd run into problems if we were using AMD, but we have no modules at all.